cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -48,178 +39,51 @@ Then('{string} radioButton on preview is {string}', (position, text) => {
-      break;
-    case 'Second':
-      radioButtonByPosition(SECOND_RADIOBUTTON).should('have.attr', 'value').should('contain', text);
-      break;
-    case 'Third':
-      radioButtonByPosition(THIRD_RADIOBUTTON).should('have.attr', 'value').should('contain', text);
-      break;
-    default: throw new Error('There are only three radio elements on the page');
-  }
+Then('{string} RadioButton has value {string}', (radioButtonName, text) => {
+  cy.radioButton(radioButtonName, 'div:nth-child(1) input').should('have.attr', 'value').should('contain', text);
 });
 
-Then('{string} RadioButton component is disabled', (position) => {
-  switch (position) {
-    case 'First':
-      radioButtonComponent(FIRST_RADIOBUTTON).should('have.attr', 'disabled');
-      radioButtonByPosition(FIRST_RADIOBUTTON).should('be.disabled')
-        .and('have.attr', 'disabled');
-      break;
-    case 'Second':
-      radioButtonComponent(SECOND_RADIOBUTTON).should('have.attr', 'disabled');
-      radioButtonByPosition(SECOND_RADIOBUTTON).should('be.disabled')
-        .and('have.attr', 'disabled');
-      break;
-    case 'Third':
-      radioButtonComponent(THIRD_RADIOBUTTON).should('have.attr', 'disabled');
-      radioButtonByPosition(THIRD_RADIOBUTTON).should('be.disabled')
-        .and('have.attr', 'disabled');
-      break;
-    default: throw new Error('There are only three radio elements on the page');
-  }
+Then('{string} RadioButton component is disabled', (radioButtonName) => {
+  cy.radioButton(radioButtonName, 'div:nth-child(1) input').should('have.attr', 'type', 'radio')
+    .and('be.disabled')
+    .and('have.attr', 'disabled');
 });
 
-Then('{string} RadioButton component is enabled', (position) => {
-  switch (position) {
-    case 'First':
-      radioButtonComponent(SECOND_RADIOBUTTON).should('not.have.attr', 'disabled')
-        .and('not.be.disabled');
-      radioButtonByPosition(FIRST_RADIOBUTTON).should('not.have.attr', 'disabled')
-        .and('not.be.disabled');
-      break;
-    case 'Second':
-      radioButtonComponent(SECOND_RADIOBUTTON).should('not.have.attr', 'disabled')
-        .and('not.be.disabled');
-      radioButtonByPosition(SECOND_RADIOBUTTON).should('not.have.attr', 'disabled')
-        .and('not.be.disabled');
-      break;
-    case 'Third':
-      radioButtonComponent(SECOND_RADIOBUTTON).should('not.have.attr', 'disabled')
-        .and('not.be.disabled');
-      radioButtonByPosition(THIRD_RADIOBUTTON).should('not.have.attr', 'disabled')
-        .and('not.be.disabled');
-      break;
-    default: throw new Error('There are only three radio elements on the page');
-  }
+Then('{string} RadioButton component is enabled', (radioButtonName) => {
+  cy.radioButtonComponent(radioButtonName).should('not.be.disabled').and('not.have.attr', 'disabled');
+  cy.radioButton(radioButtonName, 'div:nth-child(1) input').should('have.attr', 'type', 'radio')
```

Comment:
could we create a separate variable for `div:nth-child(1) input`? and use everywhere in this file

Reply:
fixed